build/yaml/crd/vpc/crd.nsx.vmware.com_subnets.yaml

```diff
@@ -63,10 +63,35 @@ spec:
                 x-kubernetes-validations:
                 - message: Value is immutable
                   rule: self == oldSelf
-              enableVLANExtension:
-                default: false
-                description: Whether this Subnet enabled VLAN extension.
-                type: boolean
+              advancedConfig:
+                description: VPC Subnet advanced configuration.
+                properties:
+                  DHCPServerAddresses:
+                    description: DHCP server IP address.
+                    items:
+                      type: string
+                    maxItems: 1
+                    type: array
```

Comment:
is this needed by new feature?

Reply:
I am not sure if specified DHCP server address need to be set in migration case when map the T1 segment to VPC Subnet. Just add the supported fields in NSX Subnet advanced_config.
@jianjuns  could you advise if this field needs to be exposed in CR?

Reply:
Yes, from subnet status API, the subnet will have specified gateway_address(the default is 172.26.1.1/28) and dhcp_server_address(the default is 172.26.1.2/28).
"results": [
{
"gateway_address": "172.26.1.3/28",
"network_address": "172.26.1.0/28",
"dhcp_server_address": "172.26.1.7/28",
"ip_address_type": "IPV4"
}
]

With my test, I found a bug (3519723) though: create a port on the subnet, the port got the same IP "172.26.1.3" as  "gateway_address" allocated.